internal/collections/map.go

```diff
@@ -41,17 +41,18 @@ func NewCaseSensitiveKeyMap(variable variables.RuleVariable) *Map {
 }
 
 func (c *Map) Get(key string) []string {
-	if len(c.data) == 0 {
```

Comment:
Wonder why we removed this.

Reply:
If c.data is empty, the c.data[key] lookup will return nil, so checking len(c.data) == 0 first wasn't required.

Reply:
true but we save the `strings.ToLower` if data is empty :)

Reply:
Ah, you're right. I'll add it back!